README.md

```diff
@@ -1,2 +1,20 @@
-# openshift-applier
-Used to apply OpenShift objects to an OpenShift Cluster
+# Welcome to the the home of the openshift-applier
+The `openshift-applier` is used to apply OpenShift objects to an OpenShift Cluster.
+
+
+### openshift-applier role
```

Comment:
I would like us to include a `/docs/` section in the new repo, and have that be "featured" as much as possible from the README.